menu.py

```diff
@@ -62,16 +62,17 @@ def update(self):
         """
         Updates the menu object, handling hover and click events.
         """
+        # somewhere I removed the code that removes the main background menu art. this is another revolutionary bandaid fix

```

Comment:
When I changed the logic of the mouse hovering over buttons stuff, I removed a line that contained the onHover() function. I think this is because the mouse is always on top of the background art menu so it's constantly being hovered, so the background ended up as a black screen